modular_skyrat/modules/modular_items/lewd_items/code/lewd_arousal/pain.dm

```diff
@@ -19,9 +20,25 @@
 			if(change_amount > 0)
 				adjustArousal(change_amount)
 			if(HAS_TRAIT(src, TRAIT_MASOCHISM))
-				var/pleasure_adjustment = change_amount / 2
-				adjustPleasure(pleasure_adjustment)
+				adjustPleasure(change_amount / 2)
 		pain += change_amount
 	else
 		pain -= abs(change_amount)
```

Comment:
I don't get it, why is this necessary? There shouldn't be any pain happening if the person has their prefs turned to off, we shouldn't be doing some operations on those variables like this.

Also the clamp() should be contained on the same line where the pain is actually adjusted, just to make it a little more efficient and prettier.